go.mod

```diff
@@ -31,6 +31,8 @@ require (
 	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
 	github.com/opencontainers/runtime-tools v0.9.0
 	github.com/opencontainers/selinux v1.10.1
+	github.com/pkg/errors v0.9.1
```

Comment:
Do we need to mention it in the go.mod?  If I understand it correctly, the errors package is needed by the `sftp` package but not directly by containers/common.

I am asking since we just got rid of the dependency and use the go-native error handling.

Reply:
I was wondering the same. I'll try removing it and see what happens